lte/gateway/docker/mme/Dockerfile.rhel8

```diff
@@ -0,0 +1,545 @@
+    cd $MAGMA_ROOT/lte/gateway/docker/mme/configs/ && \
+    echo -e '{ \n' \
+    '"realm": "magma.com",	 \n'\
+    '"use_stateless": "", \n'\
+    '"conf_dir": "/magma-mme/etc", \n'\
+    '"hss_hostname": "hss", \n'\
+    '"mcc": "001", \n'\
+    '"mnc": "01", \n'\
+    '"mmeGid": "1", \n'\
+    '"mmeCode": "1", \n'\
+    '"tac": "1", \n'\
+    '"non_eps_service_control": "OFF", \n'\
+    '"csfb_mcc": "001", \n'\
+    '"csfb_mnc": "01", \n'\
+    '"lac": "1", \n'\
+    '"s1ap_iface_name": "eth0", \n'\
+    '"s1ap_ip": "192.168.61.133/24", \n'\
+    '"s11_iface_name": "eth0", \n'\
+    '"mme_s11_ip": "192.168.61.133/24", \n'\
+    '"oai_log_level": "INFO", \n'\
+    '"remote_sgw_ip": "192.168.61.130", \n'\
+    '"attachedEnodebTacs": [], \n'\
+    '"attached_enodeb_tacs": [1] }' \
+    > mme_vars.json && \
+    jinja2 ../../../configs/templates/mme.conf.template mme_vars.json --format=json  > mme.conf
+
+# For developer's to have the same run env as in target image to debug
+# Copy the configuration file templates and mean to modify/generate certificates
+WORKDIR /magma-mme/bin
+RUN cp $C_BUILD/oai/oai_mme/mme oai_mme
+RUN cp $C_BUILD/sctpd/sctpd .
+WORKDIR /magma-mme/etc
+RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme.conf .
+RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/mme_fd.conf .
+
+# Create running dirs
+WORKDIR /var/opt/magma/configs
+# Adding mme configuration for stateful run
+RUN echo "use_stateless: false" > mme.yml
+WORKDIR /usr/local/etc/redis
+RUN cp $MAGMA_ROOT/lte/gateway/docker/mme/configs/redis_for_container.conf redis.conf
+
+WORKDIR /etc/magma
+RUN cp $MAGMA_ROOT/lte/gateway/configs/control_proxy.yml .
+RUN cp $MAGMA_ROOT/lte/gateway/configs/redis.yml .
+RUN cp $MAGMA_ROOT/lte/gateway/configs/service_registry.yml .
+
+WORKDIR /magma-mme/scripts
+RUN cp $MAGMA_ROOT/lte/gateway/c/oai/test/check_mme_s6a_certificate .
+RUN sed -i -e "s@^.*THIS_SCRIPT_PATH@#@" -e "s@\$SUDO@@" check_mme_s6a_certificate
+RUN sed -i -e "s@echo_error@echo@" -e "s@echo_success@echo@" -e "s@echo_warning@echo@" check_mme_s6a_certificate
+
+WORKDIR /magma-mme
+RUN openssl rand -out /root/.rnd 128
+
+################################################################
+# Target Image
+################################################################
+FROM registry.access.redhat.com/ubi8/ubi:latest as magma-mme
+ENV MAGMA_ROOT=/magma
+ENV C_BUILD=/build/c
+
+# Install a few tools (may not be necessary later on)
+ENV TZ=Europe/Paris
+RUN yum update && \
+    yum -y install --enablerepo="ubi-8-codeready-builder" \
+      libubsan \
+      libasan \
+      psmisc \
+      psmisc \
+      openssl \
+      net-tools \
+      tzdata && \
+    echo "/usr/local/lib" > /etc/ld.so.conf.d/local-lib.conf && \
+    echo "/usr/local/lib64" >> /etc/ld.so.conf.d/local-lib.conf && \
+    yum clean all -y && \
+    rm -rf /var/cache/yum
+
+# Copy runtime-used shared libraries from builder
+WORKDIR /lib64
+COPY --from=magma-mme-builder /lib64/libsctp.so.1 .
```

Comment:
This is the longest dockerfile I've ever seen. This sort of stuff should be handled by packages, not explicit dockerfile copies

Reply:
sorry.

When I have time, I will re-factor (it is the same on the Ubuntu 18 version)